products/phpbb.md

```diff
@@ -0,0 +1,26 @@
+---
+title: phpBB
+category: server-app
+permalink: /phpbb
+releasePolicyLink: https://www.phpbb.com/community/viewtopic.php?f=14&t=2302466
+activeSupportColumn: false
+releaseColumn: true
+releaseDateColumn: false
+releases:
+-   releaseCycle: "3.3"
+    eol: false
+    latest: "3.3.10"
+    link: https://www.phpbb.com/community/viewtopic.php?t=2635986
+-   releaseCycle: "3.2"
+    eol: 2017-05-01
+    latest: "3.2.11"
+-   releaseCycle: "3.1"
+    eol: 2016-05-01
+    latest: "3.1.12"
+-   releaseCycle: "3.0"
+    eol: 2015-05-01
+    latest: "3.0.14"
```

Comment:
It would be great to have the release date too. We could use tag dates (https://github.com/phpbb/phpbb/tags) as release dates.